### PR TITLE
Return 404 if the object is not found

### PR DESCRIPTION
**Pull request by warpech**
As it happens in many of our prefab apps, when the object with given ID is not found, the app throws an exception. With this fix, there is 404 instead of a crash.

### CODE REVIEW

File: src/CssThemer/Api/MainHandlers.cs [hunk 1]
```diff
@@ -45,8 +45,12 @@ public void Register()
             });
 
             Handle.GET("/CssThemer/partials/themes/{?}", (string objectId) => {
+                var theme = (CssTheme)DbHelper.FromID(DbHelper.Base64DecodeObjectID(objectId));
+                if (theme == null) 
+                {
+                    return 404;
+                }
```
Comment (PatrykSzwer) [inline src/CssThemer/Api/MainHandlers.cs]: Just a small note to add free space after closing bracket. Except that LGTM

File: src/CssThemer/Api/MainHandlers.cs [hunk 2]
```diff
@@ -62,6 +66,10 @@ public void Register()
             Handle.GET("/CssThemer/rendered-themes/{?}", (string objectId) =>
             {
                 var theme = (CssTheme) DbHelper.FromID(DbHelper.Base64DecodeObjectID(objectId));
+                if (theme == null) 
+                {
+                    return 404;
+                }
```
Comment (PatrykSzwer) [inline src/CssThemer/Api/MainHandlers.cs]: Same as above.

File: src/CssThemer/Api/MainHandlers.cs [hunk 3]
```diff
@@ -91,6 +99,10 @@ public void Register()
 
             Handle.GET("/CssThemer/rendered-themes-html/{?}", (string objectId) => {
                 var theme = (CssTheme)DbHelper.FromID(DbHelper.Base64DecodeObjectID(objectId));
+                if (theme == null) 
+                {
+                    return 404;
+                }
```
Comment (PatrykSzwer) [inline src/CssThemer/Api/MainHandlers.cs]: Same as above.

**Comment by warpech** [comment]
I've made some fixes according to your comments. Now with a 404 page you get this:

2017-06-28 13_49_07-cssthemer

WDYT @PatrykSzwer?

**Comment by warpech** [comment]
Thanks, merging then

**Pull request merged**